charts/values-vsphere.yaml

```diff
@@ -68,7 +68,10 @@ nodepools:
   # If you choose creation type clone a name of what you want to clone is required
   # cloneFrom: ""
 
-  # Filepath to a cloud-config yaml file to put into the ISO user-data
+  # Contents of cloud-config yaml file to put into the ISO user-data; Format should be:
```

Comment:
Can we continue using the previous format? The comment seems to state we have to use multi-line format while the example value `cloudConfig: ""` is not technically following this format.

Reply:
PR update by removing old example.